catalog/catalog.go

```diff
@@ -67,6 +67,7 @@ type PropertiesUpdateSummary struct {
 	Missing []string `json:"missing"`
 }
 
+// Optional config of CreateTable operation
```

Comment:
please use the Godoc format, i.e. start with the object name:

```suggestion
// CreateTableCfg represents the configuration used for CreateTable operations
```